src/pykeen/experiments/boxe/abboud2020_boxe_fb15k37.yaml

```diff
@@ -0,0 +1,10 @@
+metadata:
+  title: Learn FB15k-237 Dataset with BoxE as described by Abboud et al., 2020
+results:
+  hits_at_k:
+    FIXME:
```

Comment:
It *seems* to be optimistic, although I am not 100% certain about it.

https://github.com/ralphabb/BoxE/blob/584c83bb42817c67df0f70c52898aacfc916af19/BoxEModel.py#L915-L925

Maybe @ralphabb can help here :slightly_smiling_face:

Reply:
I think we can leave it to @ralphabb to curate his results ;) 

just follow the same form as the other examples in the src/pykeen/experiments directory

Reply:
I'm afraid I don't understand what you mean here. Can you re-word here?

Reply:
When we implement new models in PyKEEN, we typically curate the experimental configurations reported in their accompanying papers. For example, the R-GCN paper reported results for FB15k, so we curated the configuration (i.e., the loss used, regularization used, hyperparameters, etc.) [here](https://github.com/pykeen/pykeen/blob/master/src/pykeen/experiments/rgcn/schlichtkrull2018_rgcn_fb15k.yaml). This YAML file corresponds to the arguments that would go into the `pykeen.pipeline.pipeline()` function to directly run the same experiment to try and reproduce the results reported in the paper. 

If they did HPO, we pick the configuration for the best run, since usually papers report the best run in their main text

Reply:
Oh you I should just pass the configurations (kwargs) that align with my repo setups?

Reply:
Yup! let us know if you have any questions about how to write it

Reply:
I've written them. I'm not very sure if uniform_norm does the job (I was using uniform_ in Tester), and I couldn't find a pointer in the docs to validate the range it produces. In any case, I have clarified the intended init as a comment for us to change if this isn't currently correct.